sci-physics/pythia/pythia-8.3.12.ebuild

```diff
@@ -0,0 +1,218 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{11..13} )
+inherit toolchain-funcs python-single-r1
+
+MV=$(ver_cut 1-2)
+MY_P="${PN}${PV//./}"
+LHA_VER="6.2.1"
+
+DESCRIPTION="Lund Monte Carlo high-energy physics event generator"
+HOMEPAGE="https://pythia.org/"
+SRC_URI="test? ( lhapdf? (
+		https://lhapdfsets.web.cern.ch/lhapdfsets/current/CT10.tar.gz
+		https://lhapdfsets.web.cern.ch/lhapdfsets/current/MRST2007lomod.tar.gz
+		https://lhapdfsets.web.cern.ch/lhapdfsets/current/NNPDF23_nlo_as_0119_qed_mc.tar.gz
+		https://lhapdfsets.web.cern.ch/lhapdfsets/current/NNPDF23_nnlo_as_0119_qed_mc.tar.gz
+		https://lhapdfsets.web.cern.ch/lhapdfsets/current/cteq66.tar.gz
+		https://lhapdfsets.web.cern.ch/lhapdfsets/current/cteq6l1.tar.gz
+		https://www.hepforge.org/downloads/lhapdf/pdfsets/v6.backup/${LHA_VER}/MRST2004qed.tar.gz
+	) )
+"
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.com/Pythia8/releases"
+else
+	SRC_URI="https://pythia.org/download/${PN}${MV//./}/${MY_P}.tgz
+	$SRC_URI"
+	KEYWORDS="~amd64"
```

Comment:
Why are we dropping the `x86` keyword here?

Reply:
I was confused first now too, but rivet does not have `~x86`: https://packages.gentoo.org/packages/sci-physics/rivet

Reply:
A better solution is then to `package.use.mask` the `rivet` USE flag on `x86`. Then we can keep the x86 keyword.